src/dackar/similarity/synsetUtils.py

```diff
@@ -112,9 +108,18 @@ def semanticSimilaritySynsets(synsetA, synsetB, disambiguation=False):
 
```

Comment:
Disambiguation is missing from docstring

Reply:
removed.